src/modules/form.js

```diff
@@ -0,0 +1,95 @@
+import { creator, createModal } from './helpers';
+import { setValues, symbols } from './display';
+import {
+  getInputsValues, modifyInput, getWeather, filterData,
+} from './data';
+import { setBGImage, iconList } from './background';
+
+const radioButtons = [
+  ['kelvin', '01'], ['celsius', '02'], ['fahrenheit', '03'],
+];
+
+const units = {
+  '01': '',
+  '02': '&units=metric',
+  '03': '&units=imperial',
+};
+
+function createRadioBtn(radioContainer, radioButtons, i) {
+  const radioBox = creator(radioContainer, 'div', 'append');
+
+  const radio = creator(radioBox, 'input', 'append');
+  radio.setAttribute('type', 'radio');
+  radio.setAttribute('value', `${radioButtons[i][1]}`);
+  radio.setAttribute('id', `${radioButtons[i][0]}`);
+  radio.setAttribute('class', 'weather-input radio-input');
+  radio.setAttribute('name', 'units');
+  const label = creator(radioBox, 'label', 'append');
+  label.setAttribute('for', `${radioButtons[i][0]}`);
+  label.innerHTML = `${radioButtons[i][0].charAt(0)
+    .toUpperCase()
+    .concat(radioButtons[i][0]
+      .slice(1))}`;
+  if (i === 1) radio.checked = true;
+}
+
+function addCBToSearchBtn(button, main, units) {
+  button.addEventListener('click', async (e) => {
+    e.preventDefault();
+    const inputs = document.getElementsByClassName('weather-input');
+    const values = getInputsValues(inputs);
+    if (values) {
+      const cityName = modifyInput(values[0], units[values[1]]);
+      const data = await getWeather(cityName);
+      if (data) {
+        const allData = filterData(data);
+        setBGImage(iconList, allData[6]);
+        setValues(symbols, ...allData);
+      } else {
+        const modalContent = createModal(main);
+        modalContent.innerHTML = 'The city provided could not be found.';
+        document.getElementById('error-msg').style.display = 'block';
+      }
+    } else {
+      const modalContent = createModal(main);
+      modalContent.innerHTML = 'Fill in required field.';
+      document.getElementById('error-msg').style.display = 'block';
+    }
+  });
+}
+
+function formSearch(parent, units) {
```

Comment:
**Let's make use of the arrow function instead of using the `function` keyword**
![image](https://user-images.githubusercontent.com/31897434/92687892-3b7b3280-f334-11ea-860a-cf5e33ef18f8.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/